src/modules/filelist/icons/FileIconMime.jsx

```diff
@@ -2,23 +2,39 @@ import PropTypes from 'prop-types'
 import React from 'react'
 
 import { isDirectory } from 'cozy-client/dist/models/file'
+import flag from 'cozy-flags'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 
 import { isEncryptedFolder } from '@/lib/encryption'
 import getMimeTypeIcon from '@/lib/getMimeTypeIcon'
+import { CustomizedIcon } from '@/modules/views/Folder/CustomizedIcon'
 
 const FileIconMime = ({ file, size = 32, isEncrypted = false }) => {
   const isDir = isDirectory(file)
   const isDirEncrypted = isEncrypted || (isDirectory && isEncryptedFolder(file)) // use file.ref + file.type
 
-  return (
-    <Icon
-      icon={getMimeTypeIcon(isDir, file.name, file.mime, {
-        isEncrypted: isDirEncrypted
-      })}
-      size={size}
-    />
-  )
+  if (isDir) {
+    if (
+      flag('drive.folder-personalization.enabled') &&
+      file.decorations?.color
+    ) {
+      return (
+        <CustomizedIcon selectedColor={file.decorations?.color} size={size} />
+      )
+    }
+    return (
+      <Icon icon={getMimeTypeIcon(true, file.name, file.mime)} size={size} />
+    )
+  } else {
+    return (
+      <Icon
+        icon={getMimeTypeIcon(isDir, file.name, file.mime, {
+          isEncrypted: isDirEncrypted
+        })}
+        size={size}
+      />
+    )
+  }
```

Comment:
```suggestion
  return isDir &&
    flag('drive.folder-personalization.enabled') &&
    file.decorations?.color ? (
    <CustomizedIcon selectedColor={file.decorations.color} size={size} />
  ) : (
    <Icon
      icon={getMimeTypeIcon(isDir, file.name, file.mime, {
        isEncrypted: isDirEncrypted
      })}
      size={size}
    />
  )
```

The isEncrypted was applied anywhere in the previous version